src/account-settings/AccountSettingsPage.jsx

```diff
@@ -583,15 +583,18 @@ class AccountSettingsPage extends React.Component {
           />
           {this.renderSecondaryEmailField(editableFieldProps)}
           <ResetPassword email={this.props.formValues.email} />
-          <EditableField
-            name="year_of_birth"
-            type="select"
-            label={this.props.intl.formatMessage(messages['account.settings.field.dob'])}
-            emptyLabel={this.props.intl.formatMessage(messages['account.settings.field.dob.empty'])}
-            value={this.props.formValues.year_of_birth}
-            options={yearOfBirthOptions}
-            {...editableFieldProps}
-          />
+          {(getConfig().COLLECT_YEAR_OF_BIRTH === 'false')
```

Comment:
I believe YOB field should be added to the form if the flag is true?
```suggestion
          {(getConfig().COLLECT_YEAR_OF_BIRTH === 'true')
```